tools/dev-server.js

```diff
@@ -13,34 +13,69 @@ const { Builder } = require("./builder");
 const KARMA_PORT = 9876;
 const SERVE_PORT = 5000;
 
+class KarmaServer {
+  /**
+   * @param {string} configFile
+   * @param {string} [browser]
+   * @param {string} [grep]
+   */
+  constructor(configFile, browser, grep) {
+    this._configFile = configFile;
+    this._browsers = browser ? [browser] : undefined;
+    this._grep = grep;
+    this._isActive = null;
+  }
+
+  start() {
+    this._karmaConfig = karma.config.parseConfig(this._configFile, {
```

Comment:
is `parseConfig()` has no side effects, it might better to do this in the ctor.

Reply:
Good idea!